### DISCUSSION TITLE
feat(Select): 增加适应汉字拼音搜索项

### DISCUSSION THREAD

**Discussion by taihangyishui** [dotnetcore/BootstrapBlazor#4899] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

![image](https://github.com/user-attachments/assets/f3b4ed95-e321-4363-a03f-145128d5f287)

### Describe the solution you'd like

建议：增加拼音搜索选项； 张佬，我学习了周老师 https://blog.csdn.net/alex_zoucn/article/details/140220946这个过后；感觉在搜索前可以利用插件将汉字转成拼音，以适应搜索。请参考，不知道有用没。

**Comment by bb-auto** [comment]
@taihangyishui Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment, marked as answer]
@taihangyishui 目前组件库不内置汉语拼音相关的库，但是应该增加一个扩展让使用者更方便的扩展出自己的功能。这才是通用组件的核心

**Comment by densen2014** [comment]
组件库内置这个功能没啥必要, 就参考文章自己实现一下就行
